apps/courses/src/App.tsx

```diff
@@ -11,6 +11,8 @@ import Cookies from './components/Cookies';
 import Footer from './components/Footer';
 import { Box } from '@chakra-ui/react';
 
+import { usePerformance, SuspenseWithPerf } from 'reactfire';
```

Comment:
Make sure to remove `usePerformance` as we don't need it. Also remove the `Suspense` import for React.

Reply:
Also remove the todo from the list.